packaging/tags.py

```diff
@@ -322,7 +336,35 @@ def _is_manylinux_compatible(name, glibc_version):
 def _glibc_version_string():
     # type: () -> Optional[str]
     # Returns glibc version string, or None if not using glibc.
-    import ctypes
+    return _glibc_version_string_confstr() or _glibc_version_string_ctypes()
+
+
+def _glibc_version_string_confstr():
+    # type: () -> Optional[str]
+    "Primary implementation of glibc_version_string using os.confstr."
```

Comment:
The rest of the code base lacks docstrings for internal functions (I don't know why, but seems to be a design choice).

Reply:
Sorry, are you suggesting that this should be removed?

Reply:
Yes and no. 😄 Yes because that seems to be the style, but no because I disagree with that style choice.